Update distance.cpp

SulemanHamdani opened the pull request.

SulemanHamdani committed:
Update distance.cpp

Muhammed-Idrees-AAI commented on distance.cpp:
```diff
@@ -19,8 +19,8 @@ struct CartesianCoord
 
 	float calculateDistance(float x1, float y1, float x2, float y2)
```
you can pass two CatesianCoord here instead of individual variables.
Like:

`float calculateDistance(CartesianCoord c1, CartesianCoord c2)`

and also, this function can be taken outside the `CartesianCoord` struct.

Muhammed-Idrees-AAI commented on distance.cpp:
```diff
@@ -19,8 +19,8 @@ struct CartesianCoord
 	{
-		float distanceSquared = sqrt((pow(x2 - x1, 2)) + (pow(y2 - y1, 2)));
-		return distanceSquared;
+		float distance = sqrt((pow(x2 - x1, 2)) + (pow(y2 - y1, 2)));
```
can return it directly like:
`return sqrt((pow(x2 - x1, 2)) + (pow(y2 - y1, 2)));`

SulemanHamdani committed:
Update distance.cpp

Function CalculateDistance was removed as member function and now takes two objects as its arguments

SulemanHamdani committed:
Update distance.cpp

Added a function that calculates Chebyshev distance

SulemanHamdani committed:
Update distance.cpp
